app/security.py

```
@@ -181,6 +211,63 @@ def get_current_user(
     return user
 
 
+class PermissionChecker:
+    """
+    Dependency to check if the current user has permission for the request.
+    """
+
+    def __init__(self, tag: str):
```

Ruff (0.14.14)

[warning] Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)